server/e2e_test.go

```diff
@@ -687,7 +705,7 @@ func (h *testHarness) WaitEmptyInboxes(clients []*testClient) {
 				if len(c.inbox.packets) != 0 {
 					lines = append(lines, fmt.Sprintf("inbox #%d not empty:", c.playerNum))
 					for _, pkt := range c.inbox.packets {
-						lines = append(lines, fmt.Sprintf("  %+v\n", pkt.message.GetPacket()))
+						lines = append(lines, fmt.Sprintf("  %+v", pkt.message.GetPacket()))
```

Comment:
Why the newline removal?

Reply:
Because these are joined with `"\n"`... am I missing something?

Reply:
(that means it was either a bug before or a bug now :D)